maltd-frontend/src/components/hoc/AuthenticationGuard.js

```diff
@@ -15,6 +16,9 @@ if (process.env.REACT_APP_KEYCLOAK_REALM)
 if (process.env.REACT_APP_KEYCLOAK_CLIENT_ID)
   clientId = process.env.REACT_APP_KEYCLOAK_CLIENT_ID;
 
+if (process.env.REACT_APP_KEYCLOAK_ACCESS_ROLE)
```

Comment:
If we are adding this as an environmental variable, we should update the .env.example file to reflect the env variable that needs to be present :)

Reply:
Yup!

Reply:
Done!